### ISSUE TITLE
Only one Wiimote connecting to game

### ISSUE THREAD

**Issue by Linkstxr** [FIX94/Nintendont#1175]
Every time I launch a gamecube game with Nintendont, every controller connected to the wii disconnects, and only one controller will be able to connect, every other controller flashes but won't connect. I am using official wiimotes, on an official RVL wii and playing mario kart double dash with nintendont. (The wiimote that connects works perfectly fine) I need answers so I can play multiplayer.

**Comment by carnage702** [comment]
wiimote alone is not a vlid controller what are the other wiimotes connected to? wiimote needs an extension like classic controller and such solo wiimote is not gonna sinc with nintendont ingame.

**Comment by Linkstxr** [comment]
The player 1 controller is connected with one of those gamecube controller extensions that plug into the wiimote. The wii mote has forever lit up lights and does nothing until i plug in that extension. The player two wii mote also has one of these extensions plugged in but won't connect.

**Comment by carnage702** [comment]
there is zero garantees that 3rd party controller adapter will even work on nintendont so yeah  buy official classic controller or wiiu/switch gc adapter

**Comment by Linkstxr** [comment]
The controller works perfectly fine as I can play with it. The problem is that only one controller is able to connect, a second won't even sync into the wii when playing with nintendont. I've tried with nunchucks too, nothing will help sync in the second wiimote.

**Comment by carnage702** [comment]
> The controller works perfectly fine as I can play with it. The problem is that only one controller is able to connect, a second won't even sync into the wii when playing with nintendont. I've tried with nunchucks too, nothing will help sync in the second wiimote.
nintendont is very picky with 3rd party stuff so yeah like i said it doesnt matter if it works on wii games or other stuff, the only guarantees for nintendont is official controllers and adapters, all else is a gamble.

**Comment by carnage702** [comment]
> The controller works perfectly fine as I can play with it. The problem is that only one controller is able to connect, a second won't even sync into the wii when playing with nintendont. I've tried with nunchucks too, nothing will help sync in the second wiimote.

is the second wiimote official? is that 100% confirmed?

**Comment by Linkstxr** [comment]
No, I can play mario kart double dash with the third party controller. Once again, the problem is that the second official wii remote just simply won't connect and there is no reason for that to happen.

**Comment by Linkstxr** [comment]
Both wii remotes are official, atleast I think

**Comment by carnage702** [comment]
> Both wii remotes are official, atleast I think

easiest way to tell is check the screws, open the battery cover and there is some screws inside, if the screws are triwings(3 sides only) its official, if the screws are regular 4 sided ones its a fake/3rd party wiimote.
